config/core/test-configs.yaml

```diff
@@ -868,6 +868,14 @@ device_types:
     class: arm64-dtb
     boot_method: uboot
 
+  fvp:
+    mach: vexpress
+    arch: arm64
+    dtb: 'arm/fvp-base-revc.dtb'
+    boot_method: uboot
+    filters:
+      - passlist: {defconfig: ['defconfig']}
```

Comment:
This is going to generate standard u-boot type jobs like for a regular physical board but we need custom stuff for the FVPs - it's going to need to be a new set of templates like qemu I think. We need to provide firmware as part of the job definition (and we'll need to supply instructions for building it so kernelci can host it and provide something repeatble).